docs/document/content/user-manual/shardingsphere-proxy/startup/graalvm-native-image.cn.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "构建 GraalVM Native Image(Alpha)"
+weight = 2
++++
+
+## 背景信息
+
+本节主要介绍如何通过 `GraalVM` 的 `native-image` 组件构建 ShardingSphere-Proxy 的 `Native Image` 和对应的 `Docker Image`
+。
+
+## 注意事项
+
+- ShardingSphere Proxy 尚未准备好与 GraalVM Native Image 集成。
+  其在 https://github.com/apache/shardingsphere/actions/ 存在每日构建的任务用于测试构建。
```

Comment:
Since I haven't pushed the actual Docker image yet, I plan to add ghcr.io information in another PR.